js/index.js

```diff
@@ -397,24 +338,12 @@ addFile.onclick = async () => {
     var btnName = "rem-" + rowCount;
     element3.name = btnName;
     element3.onclick = function() {
-            removeRow(btnName);
+            utilities.removeRow(table,btnName);
```

Comment:
Small observation: use a space after a comma, makes reading code easier.

Reply:
Thank you for the feedback! @sachinkosare 😄,Will adopt good practices for code writing from now onwards.

Reply:
@sachinkosare The fixes have been merged into this [PR](https://github.com/espressif/esp-launchpad/pull/34).